check/check.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	ResourceTypeDataSource = "data source"
+	ResourceTypeFunction   = "function"
```

Comment:
This is supposed to be used with the `FileMismatchOptions` configuration option, exposed via flags like `-ignore-file-missing-data-sources`, but I missed adding a corresponding flag in the command config. I'll get that added. 

Is there a different naming convention you'd suggest for this function constant since "resource type" isn't really correct?

Reply:
a289fc8722ea21fa267c9e424a09df78b156c268 refactored the `FileMismatchOptions` struct to use a field `ResourceNames []string` rather than `Schemas map[string]*tfjson.Schema`. This helped avoid shimming function names into a `map[string]*tfjson.Schema` type. 

As best I could tell only the map key was being used in the underlying check, making the additional Schema data unnecessary, but happy to revert if that interpretation isn't correct.